src/toolbar/preview/screenshot.js

```diff
@@ -2,7 +2,7 @@ let html2canvasPromise;
 
 const screenshot = async () => {
   document.getElementById('prismic-toolbar-v2').setAttribute('data-html2canvas-ignore', true);
-  if (!html2canvasPromise) html2canvasPromise = script('https://unpkg.com/html2canvas@1.0.0-alpha.12/dist/html2canvas.min.js');
+  if (!html2canvasPromise) html2canvasPromise = script('https://html2canvas.hertzen.com/dist/html2canvas.min.js');
```

Comment:
I see where you found this link, i think using this link provide the latest version, i'm just wondering what would happen if they release a new version, it might break stuff again so maybe we should take a set version ?

Reply:
Yes, we get more bug fixes using the latest release (at the moment it's a release candidate) for version 1. 

I guess we could set it to use the release candidate and wait to see if another bug pops up

Reply:
ok 👌  thanks